DAO.sol

```diff
@@ -691,6 +700,62 @@ contract DAO is DAOInterface, Token, TokenCreation {
         paidOut[msg.sender] = 0;
         return true;
     }
+    
+    function splitSoloDAO(bool _confirm
+    ) noEther onlyTokenholders returns (bool _success) {
+        
+        // Ensure user confirms solo split and is not blocked
+        if (!_confirm || isBlocked(msg.sender))
+            throw;
+        
+        // Create new Solo DAO based off this contract
+        SplitData memory soloData;
+        soloData.newDAO = createNewDAO(msg.sender);
+        // Call depth limit reached, etc.
+        if (address(soloData.newDAO) == 0)
+            throw;
+        // should never happen
+        // Is this needed for solo split as well?
+        if (this.balance < sumOfProposalDeposits)
```

Comment:
I think to be more accurate this 'if' statement should probably be moved down a few lines and then be modified as follows:

if (this.balance < sumOfProposalDeposits + fundsToBeMoved)
    throw;